server/services/ui/shell_scripts/start_server.sh

```diff
@@ -1,5 +1,15 @@
 #!/usr/bin/env bash
 
+
+if [ $PREFECT_SERVER__GRAPHQL_URL != 'http://localhost:4200/graphql' ]
+then
+    echo "Replacing graphql references with: $PREFECT_SERVER__GRAPHQL_URL"
+    for i in /var/www/js/*.js; do
```

Comment:
Just out of curiosity how is this different from the expression we were looking at last night?

Reply:
I think the only difference from last night is that I removed the quotes around the file path, since those aren't needed.